rpa-builder/6.4/modules/ROOT/pages/common-properties.adoc

```diff
@@ -45,12 +45,12 @@ A name for the timer. RPA Builder automatically adds this property in the Ping a
 ** *Script Feedback / Reporting Function*
 *** *Record Timer*
 +
-When this option is enabled, the runtime measured for the Ping and Service Check Action Steps is passed to RPA Manager via a linked Data Source.
+When this option is enabled, the execution time measured for the Ping and Service Check Action Steps is passed to RPA Manager.
 +
 ** *Timing*
 *** *Timer Mode*
 +
-If you use Ping and Service Check Action Steps in the Control Blocks Loop, this option defines which of the Timer values measured is to be passed to RPA Manager via a linked Data Source.
+If you use Ping and Service Check Action Steps in the Control Blocks Loop, this option defines which of the Timer values measured is passed to RPA Manager.
```

Comment:
"...,this option defines which mode is used for calculating the timer value:"

More like in here https://docs.mulesoft.com/rpa-builder/6.4/toolbox-network-ping#properties

Also none of these measurement data is passed to RPA Manager.